Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM        alpine:latest
-
-COPY rds_exporter  /bin/
-# COPY config.yml           /etc/rds_exporter/config.yml
-
-RUN apk update && \
-    apk add ca-certificates && \
-    update-ca-certificates
+# Build Application
+FROM        golang:alpine as builder
+RUN         apk update && apk add git make
+COPY        . /go/src/github.com/percona/rds_exporter/
+WORKDIR     /go/src/github.com/percona/rds_exporter/
+RUN         mkdir -p /go/src/github.com/percona/rds_exporter/vendor/github.com/percona
+RUN         ln -s /go/src/github.com/percona/rds_exporter/vendor/github.com/percona/rds_exporter /go/src/github.com/percona/rds_exporter/
+RUN         git remote set-url origin https://github.com/percona/rds_exporter.git
```

Comment:
This is required in build containers to allow it to compile the binary on forks (i.e. where the git config is not pointing at percona/rds_exporter

Reply:
Can we update the project to modules to simplify the build?
There is a Jira ticket for it: https://jira.percona.com/browse/PMM-7724